services/appmotion_gateway/docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
             labels:
                 - traefik.enable=true
                 - traefik.docker.network=${PUBLIC_NETWORK}
-                - traefik.http.routers.appmotion_gateway.rule=Host(`payments.${MACHINE_FQDN}`)
+                - traefik.http.routers.appmotion_gateway.rule=${DEPLOYMENT_FQDNS_APPMOTION_CAPTURE_TRAEFIK_RULE}
```

Comment:
Careful!! this entrypoint is affects `APPMOTION_GATEWAY_PUBLIC_URL` from osparc-config `repo.config`!  
I wonder whether you can get that variable from there in here instead of creating it in the Makefile.

Reply:
This change enables us to redirect `https://pay.$URL` all products (e.g. `osparc.io` and `s4l-lite.io` on the same deployment) to the gateway. I dont think there is a changed needed in the `repo.config`.


The variable `DEPLOYMENT_FQDNS_APPMOTION_CAPTURE_TRAEFIK_RULE` is resolved *from* `osparc-config` already, *by* the Makefile.

fine for you?

Reply:
I am not sure we are on the same line. Can you please check https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/deployments/osparc-master.speag.com/repo.config?ref_type=heads#L599?

Reply:
See https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/290 should be fine in my eyes

Reply:
Excellent. thx!